src/webview/components/options-widget.tsx

```diff
@@ -52,6 +53,7 @@ export interface OptionsWidgetProps
 
 interface OptionsWidgetState {
     isTitleEditing: boolean;
+    isEnablingPeriodicRefresh: boolean;
```

Comment:
It looks like this should really be an event handler: when the value is changed to not off, we want to focus the input. Alternatively, it isn't necessary: the interval input is always available, so the user can focus it any time they want. This is different from the title editing input, whose appearance is controlled by a state value and into which we want to pass focus as soon as it appears.

Reply:
I agree with you, I thought it was nice UX but it might be a bit much, especially if the user already configured the delay to what they want once. I therefore removed the auto-focus on the input again.